front/app/containers/Admin/projects/project/participationContext/index.tsx

```diff
@@ -521,6 +536,50 @@ class ParticipationContext extends PureComponent<
               />
             )}
 
+            {participation_method === 'document_annotation' && (
```

Comment:
It looks like we are already adding code to an already bug file. I wonder if it might be better to write this piece of code as component and pass in props 🤔.

Reply:
Agreed this file is getting too big, but not going to refactor now.